AutomatedTesting/Gem/PythonTests/prefab/Prefab_BasicWorkflow_CreateAndDeletePrefab.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def Prefab_BasicWorkflow_CreateAndDeletePrefab():
+
+    CAR_PREFAB_FILE_NAME = 'car_prefab'
+    CAR_PREFAB_INSTANCE_NAME = "car_1"
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity as Entity
```

Comment:
Will try to prevent that but right now kind of can't use the same name between any two existing instances since we have a bug in prefab system. After prefab operations, prefab instance propagation will happen in the next frame and entity id we get from operations can't be used anymore. That's why we need unique names for getting correct ids. 

Here's the related ticket: https://jira.agscollab.com/browse/LYN-5831

Reply:
Fixed.